book/06-github/sections/1-setting-up-account.asc

```diff
@@ -77,13 +77,13 @@ In <<_add_email_addresses>> we can see some of the different states that are pos
 The top address is verified and set as the primary address, meaning that is where you'll get any notifications and receipts.
 The second address is verified and so can be set as the primary if you wish to switch them.
 The final address is unverified, meaning that you can't make it your primary address.
-If GitHub sees any of these in commit messages in any repository on the site, it will be linked to your user now.
+If GitHub sees any of these in commit messages in any repository on the site, it can now be linked to your user.
```

Comment:
Relatedly: apparently you authored some of these commits with a different email address which isn't currently linked to your account. 😄